javascript/exosecurity/features/support/steps-given.js

```diff
@@ -0,0 +1,35 @@
+const { defineSupportCode } = require('cucumber')
+const ExoComMock = require('../../../exocom-mock')
+const N = require('nitroglycerin')
+const portReservation = require('port-reservation')
+
+defineSupportCode(steps => {
+  steps.Given(/^an ExoCom instance$/, function(done) {
+    portReservation.getPort(
+      N(exocomPort => {
+        this.exocomPort = exocomPort
+        this.exocom = new ExoComMock()
+        this.exocom.listen(this.exocomPort)
+        done()
+      })
+    )
+  })
+  steps.Given(/^receiving this message:$/, function(message) {
+    message = JSON.parse(message)
+    message.service = this.role // eslint-disable-line no-param-reassign
```

Comment:
I think can avoid disabling this rule if we make the input parameter `messageStr` (since its a string anyway) and then message is a local variable which cause this rule to no longer apply.